assets/samplesheet.valid.csv

```diff
@@ -0,0 +1,2 @@
+sample,single_end,fastq_1,fastq_2,strandedness
```

Comment:
```suggestion
sample,fastq_1,fastq_2,strandedness
```
I think that we can assess if it's single or pair end given the presence of the paired fastq file.
Look at what the `check_samplesheet.py` script is doing there